CONTRIBUTING.md

```diff
@@ -168,7 +168,7 @@ Projects including intellectual property are only available for contributions by
 For support:
 + Submit [for help](https://stackoverflow.com/questions/tagged/fluent-web)
 + Submit [feature requests](https://fluentweb.visualstudio.com/Fluent%20Web/_workitems?_a=new&witd=Feature)
-+ Submit [a bug](https://fluentweb.visualstudio.com/Fluent%20Web/_workitems/create/Bug)
++ Submit [a bug](https://fluentweb.visualstudio.com/Fluent%20Web/_workitems/create/BuGg)
```

Comment:
looks like a typo at the end of this sentence.  Please revert this line from editing.